selfdrive/manager.py

```diff
@@ -218,7 +217,8 @@ def get_running():
   'uploader',
 ]
 
-if ANDROID:
+# TODO: if not PC:
+if not ANDROID:
```

Comment:
why did this change? seems like a debugging artifact?

Reply:
Oh, my bad. That's wrong.